vignettes/git_usage.Rmd

```diff
@@ -175,4 +175,53 @@ knitr::include_graphics("github_conflicts.png", dpi = 144)
 * [GitHub and RStudio](https://resources.github.com/whitepapers/github-and-rstudio/)
 * [Happy Git and GitHub for the useR](https://happygitwithr.com/)
 
+# Package Release
+
+## Quarterly Release
+
+A package release is done in four parts:
+
+1) Declare a freeze on all Pull Requests going into the `devel` branch via internal channels.
```

Comment:
I think we can remove this step. As we use `pre-release` for the final checks and updates, no freeze of `devel` is required.

Reply:
I'm thinking if we need to do some emergency fixes before the release having other stuff going into `devel` besides our emergency fixes might get a bit muddled.  If you we merge our fixes into pre-release then we have to remember to put them back into devel as well, which can get tedious. 

I think the freezes are nice as then all of the core developers can really focus on release activities.

Reply:
I would prefer doing the emergency fixes in `pre-release` and merging `pre-release` back into `devel` after the release. Then we do not need to rely on that everyone has seen the freeze message. It is also requires less PRs. Because if we do the emergency fixes in `devel`, we need two PRs for each fix, one for merging the feature branch into `devel` and another one for merging `devel` into `pre-release`.

Reply:
that is a good point.  i was thinking the PRs were getting a bit excessive